.vsts-pipelines/steps/test-images-linux-client.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
```

Comment:
I think this should be deleted as it isn't applicable here.

Reply:
What was the resolution to this?  I realize now my comment wasn't very clear.  I was referring to the entire yml file.  It isn't used at all therefore I think it should be removed to eliminate confusion.  I realize at some point in the future if we share the build infra it may get copied here but until that happens, I think it should not be added.

Reply:
Sorry, I wasn't thinking. I thought you were referring to the parameters not being needed, not the entire file.  But yes, this entire file can go.